arangod/Aql/DocumentProducingHelper.h

```diff
@@ -42,22 +42,30 @@ class Builder;
 class Slice;
 }
 namespace aql {
+class Expression;
 class InputAqlItemRow;
 class OutputAqlItemRow;
+class Query;
+
+enum ProjectionType : uint32_t {
```

Comment:
I'd prefer `enum class` to avoid accidental implicit casts.

Reply:
Oops, I actually wanted to use `enum class` here but it seems I failed.